content/03.methods.md

```diff
@@ -14,9 +14,36 @@ talk about generative model and how it works
 ### Word Embeddings
 mention facebooks fasttext model and how we used it to train word vectors
 ### Discriminator Model
-talk about the discriminator model and how it works
+The discriminator model is designed to improve off of the generated labels from the generative model.
+We used a convolutional neural network with multiple filters as our discriminator model.
+This network uses multiple filters with fixed widths of 300 dimensions and can have varying heights (Figure {@fig:convolutional_network}).
+For our model we each filter at a fixed height of 7, because this provided the best performance in terms of relationship classification [@arxiv:1510.03820].
```

Comment:
Is this based on the reference from the arxiv paper?

Reply:
Yeah this was based off of the arxiv paper. They did bench marking experiments using the same model I'm using for this project.